README.md

```diff
@@ -32,6 +32,28 @@ pip install dist/raydp*.whl
 ## Getting Started
 To start a Spark job on Ray, you can use the `raydp.init_spark` API. You can write Spark, PyTorch/Tensorflow, Ray code in the same python program to easily implement an end to end pipeline.
 
+### Classic Spark Word Count Example
+After we use RayDP to initialize a Spark cluster, of course we can use Spark as usual. 
+```python
+import ray
+import raydp
+
+ray.init(address='auto')
+
+spark = raydp.init_spark('word_count',
+                         num_executors=2,
+                         executor_cores=2,
+                         executor_memory='1GB')
+wordsDF = spark.createDataFrame([('look',), ('spark',), ('tutorial',), 
+                                ('spark',), ('look', ), ('python', )], ['word'])
+wordsDF.show()
+wordCountsDF = (wordsDF
+                .groupBy('word').count())
+wordCountsDF.show()
```

Comment:
add `raydp.stop_spark(..)`

Reply:
Could you add the `raydp.stop_spark(...)` in the later example as well?

Reply:
sure